From d5ce04b4b3153fff750c4342f3cd0431e0a0a401 Mon Sep 17 00:00:00 2001
From: Levi Zim <rsworktech@outlook.com>
Date: Mon, 19 Aug 2024 16:18:33 +0200
Subject: [PATCH] Fix errors in libavutil

Not actually errors. These warnings are considered as errors.
Patch-Dir: src/electron/patches/ffmpeg
Patch-Filename: 0001-Fix-errors-in-libavutil.patch
---
 libavutil/riscv/cpu.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/libavutil/riscv/cpu.c b/libavutil/riscv/cpu.c
index 4ec6d6c826..9e2b376f5e 100644
--- a/libavutil/riscv/cpu.c
+++ b/libavutil/riscv/cpu.c
@@ -18,7 +18,9 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
  */
 
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
+#endif
 #include "libavutil/cpu.h"
 #include "libavutil/cpu_internal.h"
 #include "libavutil/macros.h"
@@ -82,6 +84,7 @@ int ff_get_cpu_flags_riscv(void)
                 ret |= AV_CPU_FLAG_RV_MISALIGNED;
                 break;
             default:
+		; /* fallthrough */
         }
     }
 #elif HAVE_GETAUXVAL
-- 
2.39.2

